packages/react-server-module-tagger/package.json

```diff
@@ -4,8 +4,10 @@
   "description": "calculates module level config",
   "main": "index.js",
   "scripts": {
-    "test": "ava test.js",
-    "clean": "del index.js npm-debug.log*"
+    "lint": "xo",
+    "test": "ava test.js && xo",
+    "clean": "del index.js npm-debug.log*",
+    "watch": "chokidar index.js -c 'npm test'"
```

Comment:
"But Doug", you might say, "surely setting up a watch mode can't be this simple!", and yet it is.